ui/src/pages/motherPostNatalCare.jsx

```diff
@@ -0,0 +1,429 @@
+import {
+    Container,
+    TextField,
+    Stack,
+    Button,
+    Grid,
+    Snackbar,
+    Typography,
+    Divider,
+    useMediaQuery,
+    CircularProgress,
+    Modal,
+  } from "@mui/material";
+  import React, { useEffect, useState } from "react";
+  import { useNavigate } from "react-router-dom";
+  import Layout from "../components/Layout";
+  import { getCookie } from "../lib/cookie";
+  import Tab from "@mui/material/Tab";
+  import TabContext from "@mui/lab/TabContext";
+  import TabList from "@mui/lab/TabList";
+  import TabPanel from "@mui/lab/TabPanel";
+  import { Box } from "@mui/material";
+  import ArrowForwardIosIcon from "@mui/icons-material/ArrowForwardIos";
+  import { AdapterDateFns } from "@mui/x-date-pickers/AdapterDateFns";
+  import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
+  
+  import CurrentPatient from "../components/CurrentPatient";
+  import { apiHost, createEncounter, FhirApi } from "../lib/api";
+  import { useFormik } from "formik";
+  import * as yup from "yup";
+  import Preview from "../components/Preview";
+  import FormFields from "../components/FormFields";
+  import postnatalCareFields from "../lib/forms/motherPostNatalCare";
+
+export default function MotherPostnatalCare({ userData }) {
+    let [patient, setPatient] = useState({});
+    let [visit, setVisit] = useState();
+    let [open, setOpen] = useState(false);
+    let [loading, setLoading] = useState(false);
+    let [message, setMessage] = useState(false);
+    let [observations, setObservations] = useState([]);
+    let isMobile = useMediaQuery("(max-width:600px)");
+    const [newVisit, setNewVisit] = useState(false);
+    let [motherPostNatalCare, setmotherPostNatalCare] = useState({});
+    let [motherPostNatalCareEncounters, setmotherPostNatalCareEncounters] = useState(
+        []
+    );
+    const handleClose = () => setOpenModal(false);
+    const handleOpen = () => setOpenModal(true);
+    const [value, setValue] = useState("1");
+    let [openModal, setOpenModal] = useState(false);
+
+    const [inputData, setInputData] = useState({});
+    const [preview, setPreview] = useState(false);
+
+
+    const navigate = useNavigate();
+    
+    const fieldValues = Object.values(postnatalCareFields).flat();
+    const validationFields = fieldValues.map((item) => ({
+        [item.name]: item.validate,
+    }));
+
+    const validationSchema = yup.object({
+        ...Object.assign({}, ...validationFields),
+    });
+
+    const initialValues = Object.assign(
+        {},
+        ...fieldValues.map((item) => ({ [item.name]: "" }))
+    );
+
+    const formik = useFormik({
+        initialValues: {
+            ...initialValues,
+            facilityName: userData?.facilityName,
```

Comment:
Remove the facilityName and kmhflCode properties from the initalValues object. Not needed in this component